src/main/services/steam.service.ts

```diff
@@ -1,11 +1,12 @@
 import { UtilsService } from "./utils.service";
 import regedit from 'regedit'
-import path from "path";
+import { path, join } from "path";
```

Comment:
For me importing join here caused an error when running, removing it fixed it.
Doesn't look like it's used anyways.

Reply:
Fixed in 90d4ebc7e3d5d0e3e0c33c0d651d1597dcfa3434